frontend/packages/console-dynamic-plugin-sdk/src/extensions/horizontal-nav-tabs.ts

```diff
@@ -28,6 +28,23 @@ export type HorizontalNavTab = ExtensionDeclaration<
   }
 >;
 
+/** Contributes a tab into an HorizontalNav component matching the sectionId. */
+export type NavTab = ExtensionDeclaration<
+  'console.tab',
+  {
+    /** ContextId assigned to the horizontal nav in which the tab will be injected */
+    contextId: string;
+    /** The page to be show in horizontal tab. It takes tab name as name and href of the tab */
+    page: {
+      name: string;
+      href: string;
+    };
```

Comment:
How did we ever agree that this was the correct definition for the resource tabs?
There's no reason for nesting these properties under `page`.

@spadgett should we keep consistency with existing resource tab extension or just go ahead and make improvements now?

Reply:
I think we can deprecate page in the existing extension point and use a flatter structure, in which case we can flatten it here, too. We should make sure there is an issue tracking.

Reply:
@spadgett @christianvogt  Made the structure flat